test/mobile/test_lite_script_module.py

```diff
@@ -398,6 +403,34 @@ def forward(self, x, w):
             error_message = f"{e}"
         self.assertTrue("test_lite_script_module.py" not in error_message)
 
+    def test_source_range_raise_exc(self):
+        class FooTest5(torch.jit.ScriptModule):
+            def __init__(self, val: int):
+                super(FooTest5, self).__init__()
+                self.val = val
+
+            @torch.jit.script_method
```

Comment:
I thought these were deprecated and you're supposed to call `torch.jit.script` on the module instance.

Reply:
That's right.
The only annotations that are still ok to use are @torch.jit.ignore or @torch.jit.unused.